src/styles/main.scss

```diff
@@ -1,3 +1,232 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+$main-dark-color: #060b35;
+$main-blue-color: #00acdc;
+
+* {
+  padding: 0;
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
```

Comment:
don't write all styles in single file, create separate file for every BEM block

Reply:
done

Reply:
@ArtemTopolskyi check changes